ddtrace/internal/module.py

```diff
@@ -441,7 +436,9 @@ def uninstall(cls) -> None:
         This will uninstall only the most recently installed instance of this
         class.
         """
-        cls._check_installed()
```

Comment:
wait... was this basically not doing anything before? 0_o

Reply:
There was a refactor from @emmettbutler (I think?) that made this a no-op.